projects/fal/src/fal/auth/auth0.py

```diff
@@ -27,19 +27,17 @@ def logout_url(return_url: str):
 
 
 def _open_browser(url: str, code: str | None) -> None:
-    browser = get_browser()
-    console.print()
+    maybe_open_browser_tab(url)
```

Comment:
I actually prefer the prompt as I don't like it when cli tools randomly open pages in my browser 😅 

That might just be me. Maybe we can have a `--no-browser` flag?

Reply:
Yeah, I also don't like when it is out of nowhere, but in case of auth it seems expected since we really only do github auth anyway.

Good point about `--no-browser`, I'll take a look.